src/test/kotlin/model/WinningStatistics.kt

```diff
@@ -0,0 +1,19 @@
+package model
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class WinningStatistics {
+    @Test
+    fun `test winning amount`() {
+        val results = listOf<Int>(1, 1, 1, 1, 1)
+        assertEquals(2031555000, Statistics.calculateWinningAmount(results))
```

Comment:
```suggestion
        assertEquals(2_031_555_000, Statistics.calculateWinningAmount(results))
```

Consider using underscores every 3 digits for better readability.